lib/routes/charge/items/catalog_item.dart

```diff
@@ -60,10 +59,17 @@ class CatalogItem extends StatelessWidget {
                     mainAxisAlignment: MainAxisAlignment.start,
                     mainAxisSize: MainAxisSize.min,
                     children: <Widget>[
-                      Text(
-                        _formattedPrice(),
-                        style: theme.transactionAmountStyle,
-                      )
+                      RichText(
+                        text: TextSpan(
+                          style: theme.transactionAmountStyle,
+                          children: <InlineSpan>[
+                            _buildCurrencySymbol(),
+                            TextSpan(
+                              text: _formattedPrice(), // Price
+                            ),
```

Comment:
You can't use different font family & size for parts of text in single Text widget.
To overcome this issue symbol and price is being rendered separately under RichText.